components/Avatar/avatar.html

```diff
@@ -0,0 +1,113 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="author" content="Yash Patel" />
+    <title>Avatar</title>
+  </head>
+  <body>
+    <!-- avatar square -->
+    <h1>Avatar</h1>
+    <h1>Square avatar</h1>
+    <div class="avatar">
+      <img
+        src="https://picsum.photos/200"
+        alt="avatar"
```

Comment:
I suggest you could add different alt attributes for all images. E.g. Avatar extra small, Avatar small, Avatar large, etc.

Reply:
will do this also.